utils/src/main/scala/cromwell/pipeline/utils/UtilsModule.scala

```diff
@@ -0,0 +1,8 @@
+package cromwell.pipeline.utils
+
+import cromwell.pipeline.utils.auth.AuthUtils
+
+class UtilsModule(applicationConfig: ApplicationConfig) {
+
+  lazy val authUtils: AuthUtils = new AuthUtils(applicationConfig.authConfig)
```

Comment:
Do we need full config here or we can use a part of it?

Reply:
done